app/assets/stylesheets/base.scss

```diff
@@ -182,10 +182,6 @@ select {
   color: #4d5258 !important;
 }
 
-.ace_editor {
-  border: 2px solid #eee;
-}
-
```

Comment:
there are some leftover css classes such as .fullscreen* and similar function names in JS.

Reply:
@glekner do you want to follow up with another PR after this one gets merge?